addons/mail/models/mail_channel.py

```diff
@@ -50,15 +50,17 @@ def _get_default_image(self):
     image_128 = fields.Image("Image", max_width=128, max_height=128, default=_get_default_image)
     # members (depends=['...'] is for `test_mail/tests/common.py`, class Moderation, `setUpClass`)
     channel_partner_ids = fields.Many2many(
-        'res.partner', 'mail_channel_partner', 'channel_id', 'partner_id',
-        string='Listeners', depends=['channel_last_seen_partner_ids'])
+        'res.partner', string='Members',
```

Comment:
In your commit message you mention that:
> Due to changes in ``channel_partner_ids`` fields being a stored computed
inverse searchable field

But I don't see it being stored=True here.
Or maybe Many2many are always stored=True?
Might be nice to specify it anyway.

(Maybe I'm missing something).

Reply:
It's "stored" in `channel_last_seen_partner_ids` as per old code (where related `mail_message_mail_channel_rel` was specified). But indeed in this PR it is not really "stored" anymore, at least no more than another compute would be.

Reply:
Will update commit message, indeed.